examples/line.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import vedo
+
+vedo.settings.default_backend = "vtk"
+
+from brainrender import Scene
+from brainrender.actors import Points, Line
+
+# Display the Allen Brain mouse atlas at 25 micrometer resolution.
+scene = Scene(atlas_name="allen_mouse_25um")
+
+# Highlight the cerebral cortex; 688 is the corresponding Allen Brain Atlas ID.
+scene.add_brain_region(688, alpha=0.2, color="green")
+
+# Add two points identifying the positions of two cortical neurons.
+point_coordinates = np.array([[4575, 5050, 9750], [4275, 2775, 6100]])
+
+scene.add(Points(point_coordinates, radius=100, colors="blue"))
+
+# Display the shortest path within cortex boundaries between the two points.
```

Comment:
Is it possible to add some context about how these points were calculated? There may be an implication that this example is determining this somehow.

Reply:
For the purpose of this demo, they are points that happen to be in cortex. The specific values come from the coordinates of some cells we collected via patch-seq. What would you suggest to write?

Reply:
I meant how is the path calculated. Just something like:
```
Display the shortest path within cortex between two points (calculated using xyz)
```

Reply:
Addressed in fe49e19.